modules/apps/transfer/simulation/genesis_test.go

```diff
@@ -73,6 +73,7 @@ func TestRandomizedGenState1(t *testing.T) {
 	}
 
 	for _, tt := range tests {
+		tt := tt
```

Comment:
guess not strictly necessary but good defensive approach. There's more instances of this and it might be worthwhile going through them in a single PR. I'll open an issue.

Reply:
Indeed, I agree! Mega nit: `tc` > `tt` tho, lol